torch/_inductor/runtime/runtime_utils.py

```diff
@@ -81,35 +80,62 @@ def do_bench(fn, fn_args, fn_kwargs, **kwargs):
         return do_bench_gpu(lambda: fn(*fn_args, **fn_kwargs), **kwargs)
 
 
-def do_bench_gpu(*args, **kwargs):
+def do_bench_gpu(fn, warmup=25, rep=100, fast_flush=True):
```

Comment:
this removes some arguments, but as far as i can tell the removed args are not used in pytorch.

Reply:
Yeah, I did a quick search and the removed arguments (grad_to_none, quantiles, return_mode) were all previously unused (grad_to_none) or now unused (quantiles, return_mode). For code simplicity I opted to completely remove them.